docs/docs/tutorial-rag.md

```diff
@@ -0,0 +1,287 @@
+# Example: Model Based Evaluation on RAG applications
+
+Retrieval Augmented Generation (RAG) is a common approach of building Generative AI applications that have access to custom knowledge bases. 
+
+In this example, we'll show an example that has a retrieval step to get documents. By tracking this, you can debug your app and see what documents were pulled into the LLM context.
+We'll also show how to evaluate it using an LLM judge.
+
+## Build a knowledge base
```

Comment:
@scottire the sections are very nice and clear - adding some numbers to them 1, 2a, 2b, etc. with a first index/summary would help me to put the steps together